man/systemd-tmpfiles.xml

```diff
@@ -134,6 +134,13 @@
         removed.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--user</option></term>
+        <listitem><para>Execute "user" configuration, i.e. tmpfiles.d files
+        in user configuration directories. In this mode, configuration lines
+        are allowed to include the %U/%u/%h specifiers.</para></listitem>
```

Comment:
%U → \`%U\` maybe?

Reply:
Actually this sentence is obsolete. I'll remove it.